### ISSUE TITLE
Can't Place Explosives/Sandbags ETC.

### ISSUE THREAD

**Issue by IFaultline** [acemod/ACE3#1677]
ACE 3.1.1.0 Stable

**Mods**
*@CBA_A3*
*@Task_Force_Radio*
*@ace*

**Placed Modules**
 * ACE_medical_System*
* ACE_Advanced_medical_System*
* ACE_advanced_Ballistics*
* ACE_Wind_Deflection*

**Description:**
*We play on our own Server and Noone can place Explosives or Sandbags, we tried every Class ingame. The Menü pops up (Place;rotate) and if we Click place the character just shoots, Did i forget a module or something ?*

**Steps to reproduce:**
* Take Explosives, try to Place them.*

**Where did the issue occur?**
*"Multiplayer"*

**Comment by esteldunedain** [comment]
Please post the RPT log of the session.

**Comment by IFaultline** [comment]
http://pastebin.com/5UXBpu5Z

I forgot to add some mods above, but it also didn't worked when i turned them off

btw:sry for my english, i'm german^^

**Comment by PabstMirror** [comment]
Could you check the controls for `fire` (Vanilla Arma Control config)

![untitled](https://cloud.githubusercontent.com/assets/9376747/8236339/483de1d0-15ad-11e5-8fe8-ab1de35f4cc5.jpg)

And see what is there, and what order it is in.

**Comment by esteldunedain** [comment]
I'm pretty sure this is due to one of the MANY addons loaded, or possibly mission errors (many of those too).

>  forgot to add some mods above, 

Certainly an understatement

> but it also didn't worked when i turned them off

We'll, please provide a full RPT of a session with only CBA+ACE in which this fails.

**Comment by IFaultline** [comment]
alright i found the problem : MODS

gonna try to activate the mods one by one to find the bad mod
sry for the waste of time

**Comment by esteldunedain** [comment]
> gonna try to activate the mods one by one to find the bad mod
> sry for the waste of time

Nah, no problem, don't worry about it. But please, once you find out what's causing the problem please report it back here.

Thanks

**Comment by IFaultline** [comment]
uhm again sorry ^^ i made a mistake... i testet it in the Editor with all mods activated, it worked,
the error is caused by the current Mission we are running [Battlezone Altis (Domination)] Gonna find the error in the mission scripts by myself..
anyway thanks for the help :)

**Comment by shadowmouse** [comment]
I ran into this issue when developing my latest mission and was able to isolate the issue. ACE appears to use some functionality related to the addAction command to attach the listener for the mouse click to place explosives and sandbags. Any use of the removeAllActions command on the player breaks this connection. I got around it by moving my interface into the Commanding Menu (Communications [8]) so actions weren't added to the scroll menu.
